heltour/tournament/tasks.py

```diff
@@ -438,6 +438,8 @@ def do_generate_pairings(sender, round_id, overwrite=False, **kwargs):
 @app.task()
 def validate_registration(reg_id):
     reg = Registration.objects.get(pk=reg_id)
+    reg.last_validation_try = timezone.now()
```

Comment:
Probably best to rewrite this as an update, rather than a `.save()`  Something like:
`Registration.objects.filter(pk=reg_id).update(last_validation_try=timezone.now()`

This means that ONLY the last_validation_try will be updated at this point, which is slightly safer. Practically it probably doesn't make any difference, but it's a good habit to have.